src/metrics_server/connection_metrics.ts

```diff
@@ -51,6 +52,7 @@ function getConnectionRowsFromReport(report: HourlyConnectionMetricsReport): Con
       endTimestamp: endTimestampStr,
       userId: userReport.userId || undefined,
       bytesTransferred: userReport.bytesTransferred,
+      tunnelTimeSec: userReport.tunnelTimeSec || undefined,
       countries: userReport.countries || [],
```

Comment:
Not for this PR, but we could upgrade the code to first take a userReport.country, then fallback to userReport.countries. Though we would have to change the Bigtable code as well

Reply:
Ack.